src/domain/Amigos/Amigos.tsx

```diff
@@ -32,17 +31,27 @@ class Amigos extends Component {
     msjToast: "",
     estaBuscando: false,
     idAmigoAEliminar: null,
+
+    // Obtenemos el identificador del usuario del contexto
+    // de autenticación, lo asignamos al estado
+    idUsuario: this.context.user?.IdUsuario ?? null,
   };
 
   DURACION_TOAST = 3000;
 
-  // const amigosCtx = useContext(AmigosContext);
-  // const [estadoIniElim, setIniciarEliminacion] = useState(false);
-  // const [msjToast, setMsjToast] = useState("");
-  // const [estaBuscando, setEstaBuscando] = useState(false);
+  private __obtenerAmigo(id: null) {
```

Comment:
¿Porqué id null? ¿Se podría especificar de mejor manera el tipo de dato?

Reply:
Si. Puede ser número o null.
```suggestion
  private __obtenerAmigo(id: number | null) {
```